io/common/bootlist_test.py.data/bootlist_test_disk.yaml

```diff
@@ -0,0 +1 @@
+disks: "None"
```

Comment:
Is this right ?

Reply:
ya, i make default as None so i written here.

Reply:
ok